code/modules/holodeck/basketball.dm

```diff
@@ -0,0 +1,107 @@
+//Basketball items and structures
```

Comment:
maybe a little nitpicky, but do you think it's worth moving these out of the `code/modules/holodeck` folder since they're not holodeck only items

Reply:
Agreed, I had intended to do this and forgot.